deployment/volumes/volume-snapshots/snapshot.yaml

```diff
@@ -2,7 +2,7 @@
   apiVersion: snapshot.storage.k8s.io/v1alpha1
   kind: VolumeSnapshot
   metadata:
-    name: uploads-snapshot
+    name: YYYY-MM-DD-uploads-snapshot
```

Comment:
can you still `apply` this file? or would it need to be updated with the year, month and day?

Reply:
Each snapshot has a timestamp when it has been created, so is this necessary? For uniqueness, you could just ask the user to pick a different name

Reply:
@Tirokk brought up that it makes it easier to see what day it was created by prepending the date. With the timestamp, you can easily see how many days ago it was created, but why not make it obvious what day, month, year? Also, this is an improvement over `uploads-claim-{n}`, don't you think?

Reply:
This have to be edited before one `apply` for sure.